enrolments/tests/utils/test_load_initial_data.py

```diff
@@ -62,7 +57,6 @@ def test_load_initial_data(
             verbose=False,
         )
 
-        mock_create_fields.assert_called_once()
```

Comment:
since we're removing the mock, can we ensure the # of fields matches what we expect?